src/game/ui/SettingsMenu.cxx

```diff
@@ -31,9 +34,10 @@ void SettingsMenu::draw() const {
   ui::SetNextWindowPos(ImVec2((screenSize.x - windowSize.x)/2, (screenSize.y - windowSize.y)/2));
   ui::SetNextWindowSize(windowSize);
 
-  const ImVec2 buttonSize(200, 40);
-  const ImVec2 buttonOffset((windowSize.x - buttonSize.x) / 2, buttonSize.y / 2);
-  const ImVec2 widgetSize((windowSize.x / 2) - 8, 20);
+  const float btnh = 20.f * io.FontGlobalScale;
```

Comment:
Can you rename the variable to be a bit more explanatory? I'm not sure what exactly this does/is from the name.